### PR TITLE
Typescript types

### PR DESCRIPTION
**Pull request by sniper304**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
-module.exports = {
+import { TranslationMessages } from 'ra-core';
```
Comment (koresar) [inline src/index.ts]: `import type` is better perhaps?

Comment (sniper304) [reply src/index.ts]: Yeah, fixed, thanks.

**Comment by koresar** [comment]
I'd leave for @sorjef to review and publish to NPM. Because I'm not the RA user myself.

**Comment by sniper304** [comment]
@koresar Still no review...
@sorjef Please review when you will have a free time.

**Comment by koresar** [comment]
@sniper304 could you please elaborate on why we need TS in this tiny little package? I am mostly interested in: how exacly TS helps the module users?

**Comment by sniper304** [comment]
@koresar With typescript module user will be easily replace by correct type some translation files if it needed. Also if we can compare with standard translation for English locale (which also strongly typed), for correct using this module in TS - we need to re-declare this package as module and only then ts will ignore types.

**Comment by koresar** [comment]
Added missing strings: https://github.com/koresar/ra-language-ukrainian/commit/3e5b36e2edd1851f74c2d7b6d62c57af5c42db37

While adding I realised why TS is necessary. Thank you for the PR!

**Comment by koresar** [comment]
Published as v4: https://github.com/koresar/ra-language-ukrainian/releases/tag/v4.0.0

**Pull request merged**